CHANGELOG.md

```diff
@@ -11,6 +11,7 @@
   - `sequence`
 - Added support for registering and calling Stored Procedures with `TABLE` return type.
 - Added support for parameter `length` in `StringType()` to specify the maximum number of characters that can be stored by the column.
+- Added support to run on python 3.9 runtime environment.
 - Added the alias `functions.element_at()` for `functions.get()`.
 - Set default reviewer.
```

Comment:
Is this here by accident?

Reply:
This will be removed. It was here by accident